src/components/CheckoutLogin/CheckoutAsGuest.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Link } from "react-router-dom";
+import { Button, OverlayTheme, OverlayType } from "..";
+import { OverlayContextInterface } from "../Overlay";
+
+const CheckoutAsGuest: React.FC<{
+  overlayContext: OverlayContextInterface;
+  checkoutUrl: string;
+}> = ({ overlayContext, checkoutUrl }) => (
+  <div className="checkout-login__guest">
+    <h3 className="checkout__header">Continue as a guest</h3>
+    <p>
+      If you don’t want to register you account at our store don’t worry. You
+      can finish your checkout as a guest. You’ll be treated just as good as a
```

Comment:
Avoid using passive phrases 👍 

```suggestion
      can checkout as a guest. We care about you just as good as any
```